### DISCUSSION TITLE
SentryClips and SavedClips not archiving

### DISCUSSION THREAD

**Discussion by alexmereness** [marcone/teslausb#795] [Q&A] [answered]
I got TeslaUSB up and running, but I'm hitting a small snag. TeslaUSB is able to find and successfully transfer files to my archive server (Windows file share), however, the only directory it's transferring is "RecentClips" (and a .teslausbtmp directory). It's not transferring the other two directories, "SentryClips" or "SavedClips". I know there are clips in those directories on the Raspberry Pi because I can see them from the TeslaUSB web interface as well as in my Telsa. For some reason they're just not being archived. Any ideas on what might be happening or how to fix? Interestingly enough, the very first time I set this up I didn't have this issue...I re-imaged my device and since then have had the problem. Any help would be greatly appreciated. 

**Hardware**
-Pi Zero 2W
-128gb Samsung Pro Endurance Micro SD Car
-2018 Tesla Model 3 (AWD-LR)

Archive Server: Windows 10 w/network file share

**Archive Log (Most recent snippet)**
Sun 22 Oct 17:34:31 MDT 2023: Archive is reachable.
Sun 22 Oct 17:34:31 MDT 2023: Trying to set time...
Sun 22 Oct 17:34:31 MDT 2023: Time adjusted by 0.244493 seconds after 0.230000 seconds
Sun 22 Oct 17:34:31 MDT 2023: Not keeping car awake.
Sun 22 Oct 17:34:53 MDT 2023: taking snapshot of cam disk in /backingfiles/snapshots/snap-000053
Sun 22 Oct 17:34:56 MDT 2023: took snapshot
Sun 22 Oct 17:34:57 MDT 2023: comparing new snapshot with /backingfiles/snapshots/snap-000052/snap.bin
Sun 22 Oct 17:34:57 MDT 2023: making links for /tmp/snapshots/snap-000053, retargeted to /backingfiles/snapshots/snap-000053/mnt
Sun 22 Oct 17:35:02 MDT 2023: made all links for /tmp/snapshots/snap-000053
Sun 22 Oct 17:35:02 MDT 2023: Archiving...
Sun 22 Oct 17:35:02 MDT 2023: Ensuring cam archive is mounted...
Sun 22 Oct 17:35:02 MDT 2023: Mounting /mnt/archive...
Sun 22 Oct 17:35:03 MDT 2023: Mounted /mnt/archive.
Sun 22 Oct 17:35:03 MDT 2023: Ensured cam archive is mounted.
Sun 22 Oct 17:35:03 MDT 2023: Checking saved folder count...
Sun 22 Oct 17:35:10 MDT 2023: There are 8 event folder(s) with 2470 file(s) and 0 track mode file(s) to move.  8 short recording(s) will be skipped.
Sun 22 Oct 17:35:10 MDT 2023: Starting recording archiving
Sun 22 Oct 17:35:10 MDT 2023: Archiving 2470 file(s) including 8 event folder(s) starting at Sun 22 Oct 17:35:10 MDT 2023
Sun 22 Oct 17:45:31 MDT 2023: waiting up to 90 seconds for idle interval
Sun 22 Oct 17:45:46 MDT 2023: write in progress
Sun 22 Oct 17:45:47 MDT 2023: no longer writing, wrote 26157056
Sun 22 Oct 17:45:48 MDT 2023: going back to writing state
Sun 22 Oct 17:45:54 MDT 2023: no longer writing, wrote 84795392
Sun 22 Oct 17:46:00 MDT 2023: no writes seen in the last 5 seconds
Sun 22 Oct 17:46:00 MDT 2023: taking snapshot of cam disk in /backingfiles/snapshots/snap-000054
Sun 22 Oct 17:46:04 MDT 2023: took snapshot
Sun 22 Oct 17:46:04 MDT 2023: comparing new snapshot with /backingfiles/snapshots/snap-000053/snap.bin
Sun 22 Oct 17:46:04 MDT 2023: making links for /tmp/snapshots/snap-000054, retargeted to /backingfiles/snapshots/snap-000054/mnt
Sun 22 Oct 17:46:09 MDT 2023: made all links for /tmp/snapshots/snap-000054

**Comment by marcone** [comment, marked as answer]
You're not (yet) seeing SentryClips and SavedClips being archived because it's still archiving the RecentClips. You have a total of 2470 files to archive, of which probably over 2000 are RecentClips. It's going to take quite a while to archive all that (possibly hours), and it archives in alphabetical order, so it starts with the RecentClips.
